docs/node.md

```diff
@@ -38,13 +38,15 @@ must have appropriate conditions, see below.
 
 ### Node Condition
 Node Condition describes the conditions of `Running` nodes. Current valid
-conditions are `NodeReachable` and `NodeReady`. In the future, we plan to
-add more. `NodeReachable` means the node can be reached within the cluster.
-`NodeReady` means the kubelet returns StatusOK for HTTP health check. Different
-condition provides different level of understanding for node health. Kubernetes
-will make a comprehensive scheduling decision based on the information. Node
-condition is represented as a json object. For example, the following conditions
-mean the node is reachable from its cluster, but not ready to accept pods:
+conditions are `NodeReachable`, `NodeReady` and `NodeSchedulable`. In the
+future, we plan to add more. `NodeReachable` means the node can be reached
+within the cluster. `NodeReady` means the kubelet returns StatusOK for HTTP
+health check. `NodeSchedulable` means node is ready to schedule any new
+pods. Different condition provides different level of understanding for node
+health. Kubernetes will make a comprehensive scheduling decision based on the
+information. Node condition is represented as a json object. For example, the
+following conditions mean the node is reachable from its cluster, node is in
+sane state but not ready to accept pods:
```

Comment:
Isn't {Schedulable: Full, Ready:None} the conditions for what your are describing here?